packages/framework/src/Model/Product/Availability/ProductAvailabilityFacade.php

```diff
@@ -5,33 +5,32 @@
 namespace Shopsys\FrameworkBundle\Model\Product\Availability;
 
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\LocalCache\LocalCacheFacade;
 use Shopsys\FrameworkBundle\Component\Setting\Setting;
 use Shopsys\FrameworkBundle\Component\Translation\Translator;
 use Shopsys\FrameworkBundle\Model\Product\Product;
 use Shopsys\FrameworkBundle\Model\Stock\ProductStockFacade;
 use Shopsys\FrameworkBundle\Model\Store\StoreFacade;
-use Symfony\Contracts\Service\ResetInterface;
 
-class ProductAvailabilityFacade implements ResetInterface
+class ProductAvailabilityFacade
 {
     protected const DAYS_IN_WEEK = 7;
 
-    /**
-     * @var array<string, bool>
-     */
-    protected array $productAvailabilityDomainCache = [];
+    protected const string PRODUCT_AVAILABILITY_CACHE_KEY = 'productAvailabilityDomainCache';
```

Comment:
This should be named `PRODUCT_AVAILABILITY_CACHE_NAMESPACE`

Reply:
thx